dll/win32/shell32/lang/en-US.rc

```diff
@@ -925,6 +925,9 @@ BEGIN
     IDS_CANTDISCONNECT "Unable to disconnect (Error Code: %lu)."
     IDS_NONE "(None)"
 
+    IDS_EXPAND "Exp&and"
+    IDS_COLLAPSE "Coll&apse"
```

Comment:
Just to be sure: there is either one or the other string being displayed, but not both at the same time? (I'm asking because of the keyboard accelerator.)

Reply:
Yes, a tree item cannot be expanded and collapsed at the same time.